scripts/cli.sh

```diff
@@ -90,7 +90,7 @@ rm -f help_general.md help_bn.md help_vc.md help_am.md help_vm.md help_vm_create
 
 # only exit at the very end
 if [[ $changes == true ]]; then
-    echo "Exiting with error to indicate changes occurred..."
+    echo "Exiting with error to indicate changes occurred. Run `./scripts/cli.sh` and commit the changes to fix."
```

Comment:
Nice!

A few things to note:
1. This hint will be displayed both in CI and locally. So users running it locally will get the hint to run it again even when the changes have been applied. Running a 2nd time will not display the hint though, and given it takes a very small amount of time to run, probably isn't a big deal, but thought I'd mention it here for completeness. 
1. This won't work for Windows users. They will have to use `make cli`.
1. This will contradict the comment on line 4. It was originally added to ensure that the relative paths we were using would be correct, but we should remove it. So long as the users follow the hint's instructions it should be fine.

Reply:
Thanks for the feedback!

I've made a change and added `make cli-local` (as suggested by @jimmygchen). That should cover points (2) and (3).

(1) is still valid but I'm not sure it's worth the effort to fix. I'm happy to merge with (1) outstanding, LMK what you think.